pdaltools/add_points_in_pointcloud.py

```diff
@@ -142,10 +139,9 @@ def add_points_to_las(
     z_coords = input_points_with_z.geometry.z
     classes = virtual_points_classes * np.ones(len(input_points_with_z.index))
 
-    with laspy.open(input_las, mode="r") as las:
-        las_data = las.read()
+    # Open the input LAS file to check and possibly update the header
```

Comment:
détail : en fait tu ne mets pas à jour le header de l'entrée (et c'est normal, on ne modifie pas notre entrée), du coup je verrais bien qqc comme 
```suggestion
    # Open the input LAS file to check and possibly update the header of the output
```